### ISSUE TITLE
Add counterpart for ARIA roles on controls etc

### ISSUE THREAD

**Issue by rogerhutchings** [zooniverse/Panoptes-Front-End#3183]
We'll need to use Counterpart's `translate()` method as they have to be plain strings - https://github.com/martinandert/counterpart#lookup

**Comment by eatyourgreens** [comment]
I think I'm missing something here. ARIA role and state attributes don't need translation, they're taken from [a fixed list of values](http://karlgroves-sandbox.com/CheatSheets/ARIA-Cheatsheet.html).

**Comment by rogerhutchings** [comment]
huh, TIL
